frontendUiTableImpl/src/main/java/com/virtuslab/gitmachete/frontend/ui/impl/cell/BranchOrCommitCellRendererComponent.java

```diff
@@ -242,6 +250,19 @@ private static String getLabel(SyncToRemoteStatus status) {
         Case($(DivergedFromAndOlderThanRemote), "  (diverged from & older than ${remoteName})"));
   }
 
+  private static String getSyncToParentStatusBasedTooltip(IGitMacheteNonRootBranch currentBranch) {
+    var currentBranchName = currentBranch.getName();
+    var parentBranchName = currentBranch.getParentBranch().getName();
+    return Match(currentBranch.getSyncToParentStatus()).of(
+        Case($(InSync), "Branch '${currentBranchName}' is in sync with '${parentBranchName}'"),
+        Case($(InSyncButForkPointOff), "Branch '${currentBranchName}' is in sync with '${parentBranchName}', " +
+            "but the range of commits belonging to '${currentBranchName}' is uncertain; consider overriding fork point from right-click menu"),
+        Case($(OutOfSync), "Branch '${currentBranchName}' is out of sync with '${parentBranchName}'; " +
+            "consider rebasing onto '${parentBranchName}' from right-click menu"),
+        Case($(MergedToParent), "Branch '${currentBranchName}' is merged to '${parentBranchName}'; " +
+            "consider sliding out from right-click menu"));
```

Comment:
Nit: it sounds like a bit like the item is supposed to be slide out from the menu ;)

Reply:
So maybe sth like  `consider selecting 'Slide out' in right-click menu` :thinking: